etl/pad/raw/skills/emoji_en/active_skill_text.py

```diff
@@ -0,0 +1,546 @@
+from pad.raw.skills.en.skill_common import *
```

Comment:
You probably don't need this file or the leader skill one given that they're just copies of their text counterparts.

Reply:
done